src/lib/tracking.ts

```diff
@@ -0,0 +1,34 @@
+import { EpicMetadata } from '../components/ContributionsEpic';
+
+type LinkParams = {
+    REFPVID: string;
+    INTCMP: string;
+    acquisitionData: string;
+};
+
+export const getTrackingUrl = (baseUrl: string, params: EpicMetadata): string => {
+    const acquisitionData = encodeURIComponent(
+        JSON.stringify({
+            source: params.platformId,
```

Comment:
Just to check - by default if you make a POST request and any of these params are missing, they'll be empty in the resulting button URL right?

Reply:
Good point - we don't do anything fancy if one the values is missing as `JSON.stringify` will omit any key/value pairs where `value` is `undefined`. This applies to everything inside the `aquisitionData` object and Frontend applies the same logic. But I did make a little tweak here, to make sure `REFPVID` defaults to `not_found` (Frontend does the same) and `INTCMP` defaults to empty string (Frontend would allow `undefined` but I don't think we should replicate that).